xAPI.md

```diff
@@ -306,10 +306,21 @@ be unintuitive and/or lengthy to dissect into a list of requirements.
 
 ## 4.0 Binding to JavaScript Object Notation (JSON)
 
+This specification makes a specific binding to JSON for most of the data structures allowed.  There are some 
+generic document portions, but JSON is the current structure for Statements and the recommended use for document 
+types.  While other bindings of xAPI could potentially exist, this document does not cover such cases.
+
 ### 4.1 Serialization
 
+JSON data structures can use multiple serializations in xAPI. This means that properties and objects can have different orderings  (but not different parent/child relationships) and still be equivalent. A Statement's serialization will not actually change the Statement in terms of <a href="#statement-immutablity-and-exceptions"> immutability </a>, in some cases, even when the values change.  Statements that are <a href="#signature"> signed </a>are expected to maintain their original serialization before undergoing the signature process.
```

Comment:
I suggested on the call that this section should be mostly generic about json, possibly linking to sections of the spec where it's relevant.